.github/workflows/test_models_dafny_verification.yml

```diff
@@ -95,19 +56,11 @@ jobs:
           distribution: "corretto"
           java-version: "17"
 
-      - name: Generate Polymorph Wrapper Dafny code
-        working-directory: ./${{ matrix.library }}
-        run: |
-          make polymorph_dafny DAFNY_VERSION_OPTION="--dafny-version $DAFNY_VERSION"
-
-      - name: Verify ${{ matrix.library }} Dafny code
-        working-directory: ./${{ matrix.library }}
-        run: |
-          # This works because `node` is installed by default on GHA runners
-          CORES=$(node -e 'console.log(os.cpus().length)')
-          make verify CORES=$CORES
-
-      - name: Check solver resource use
-        working-directory: ./${{ matrix.library }}
-        run: |
-          make dafny-reportgenerator
+      - name: Execute smithy-dafny-codegen-test tests
+        uses: gradle/gradle-build-action@v2
+        env:
+          JUNIT_SHARD: ${{ matrix.shard }}
+          JUNIT_SHARD_COUNT: ${{ inputs.num_shards }}
+        with:
+          arguments: :smithy-dafny-codegen-test:test --tests '*smithydafny*' --info
```

Comment:
Does this gradle build action trigger Dafny verification?

Or is it just running tests?

The command is not clear.